### PR TITLE
Override the jackson-databind that gets pulled in by avro.

### PR DESCRIPTION
**Pull request by arvindth**
- Verified that the resulting tarball no longer contained incorrect jackson-databind versions
  - Also upgrade to avro 1.9.1 which has some desired bugfixes

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -168,6 +168,14 @@ if (false) {
...
 
 subprojects {
+
+  // enable running :dependencies task recursively on all subprojects
+  // eg: ./gradlew allDeps
+  task allDeps(type: DependencyReportTask) {}
+  // enable running :dependencyInsight task recursively on all subprojects
+  // eg: ./gradlew allDepInsight --configuration runtime --dependency com.fasterxml.jackson.core:jackson-databind
+  task allDepInsight(type: DependencyInsightReportTask) doLast {}
+
```
Comment (ijuma) [inline build.gradle]: Can we submit this change to Apache Kafka?

Comment (arvindth) [reply build.gradle]: Sure, I can put it put it up there. Should I remove it from this one?

Comment (arvindth) [reply build.gradle]: Removed from this PR. I'll put up a separate PR against upstream.

**Comment by cmccabe** [comment]
Basically, it looks like the problem is that the original fix didn't make it to ccs-kafka, just ce-kafka.  The original fix was:
```
commit f0e08de464fae69a332338f87fcde6a37355d82c
Author: Ismael Juma <ismael@juma.me.uk>
Date:   Thu Oct 17 09:50:51 2019 -0700

    MINOR: Fix jackson version inconsistency; update netty, boringssl, avro (#829)
...
```

So, comparing this to the original fix, why are we still using avro 1.9.0 when we're using avro 1.9.1 in ce-kafka?  avro is 1.9.1 in `common/pom.xml`.

**Comment by ijuma** [comment]
Yeah, we should probably update avro to 1.9.1 here too.

**Comment by arvindth** [comment]
@cmccabe I can certainly make the same changes but I don't have the full context for those. Are you suggesting I replicate all those changes? Or just the avro version update.

**Comment by cmccabe** [comment]
Thanks, @arvindth.  I was suggesting just the avro version update.  The other changes are not relevant to ccs-kafka, since it doesn't have a netty dependency (for example).

**Comment by cmccabe** [comment]
LGTM

**Comment by cmccabe** [comment]
backported to ccs-2.4 as well

File: build.gradle (hunk 1 above)
Comment (arvindth) [reply build.gradle]: apache/kafka#7694

**Pull request merged**